src/Hedgehog/Extras/Test/File.hs

```diff
@@ -273,14 +277,25 @@ assertIsYamlFile fp = GHC.withFrozenCallStack $ do
     Right _ -> return ()
     Left msg -> H.failMessage GHC.callStack msg
 
--- | Checks if all files gives exists. If this fails, all files are deleted.
+-- | Asserts that the given file exists.
+assertFileExists :: (MonadTest m, MonadIO m, HasCallStack) => FilePath -> m ()
+assertFileExists file = do
```

Comment:
I'm wondering, if shouldn't the callstack be frozen here? Otherwise I think we will get line number to this file, instead of the function usage site.

Reply:
If only it wasn't so easy to forget 😂